### ISSUE TITLE
Sort by profit/prices

### ISSUE THREAD

**Issue by Blixtmen** [felipelincoln/albion-blackmarket-tool#7]
Any plans to add sorting functionality?
Would be very handy to have the highest profits on top of the list! Great program either way.

**Comment by felipelincoln** [comment]
Yeah, sorting was the next thing I was to implement, but I'm not sure if this repo will be maintained anymore since the albion-data-client servers are going to be deactivated soon (or has already been). :face_with_head_bandage:
